tools/migration/export

```diff
@@ -0,0 +1,136 @@
+
+class Project:
+    def __init__(self, project_id, name, public, members):
+        self.project_id = project_id
+        self.project_name = name
+        self.public = public
+        self.members = members
+
+class ProjectMember:
+    def __init__(self, user_id, username, role):
+        self.user_id = user_id
+        self.username = username
+        self.role = role
+
+class HarborUtil:
+    def __init__(self, dbuser, dbpwd):
+        self.serverName = 'localhost'
+        self.user = dbuser
+        self.password = dbpwd
+        self.port = '3306'
+        self.subDB = 'registry'
+        self.db = None
+        self.cursor = None
+
+    def connect(self, times=1):
+        try:
+            self.db = MySQLdb.connect(host=self.serverName, user=self.user,
+                                      passwd=self.password, db=self.subDB)
+            self.cursor = self.db.cursor()
+        except Exception, e:
+            print str(e)
+
+    def close(self):
+        try:
+            self.cursor.close()
+            self.db.close()
+        except Exception, e:
+            print e
+
+    def get_users(self):
+        try:
+            query = "SELECT user_id, username, sysadmin_flag from registry.user where deleted=0"
+            self.cursor.execute(query)
+            self.cursor.fetchall()
+            for result in self.cursor:
+                userTemp = User(int(result[0]), result[1], result[2])			
+                users.append(userTemp)																	 	
+        except Exception, e:
+            print e
+
+    def get_projects(self):
+        try:
+            query = "SELECT project_id, name, public from registry.project where deleted=0"
+            self.cursor.execute(query)
+            self.cursor.fetchall()
+            for result in self.cursor:
+                members = self.get_members_by_project_id(result[0])			
+                projectTemp = Project(int(result[0]), result[1], result[2], members)
+                projects.append(projectTemp)								
+        except Exception, e:
+            print e
+
+    def get_members_by_project_id(self, project_id):
+        members_of_project = []	
+        try:
+            query = "select u.user_id,u.username,r.name as rolename from registry.user u join registry.project_member pm on pm.project_id=%s and u.user_id = pm.user_id join registry.role r on pm.role = r.role_id where u.deleted=0" % project_id
+            self.cursor.execute(query)
+            self.cursor.fetchall()
+            for result in self.cursor:
+                member = ProjectMember(int(result[0]), result[1], result[2])		
+                members_of_project.append(member)		   	
+            return json.dumps([member.__dict__ for member in members_of_project])
+        except Exception, e:
+            print e
+				
+
+def main():
+    commandline_input = Parameters()
+    harbor = HarborUtil(commandline_input.dbuser, commandline_input.dbpwd)
+	
+    try:
+	    harbor.connect()
```

Comment:
if any of the following call fails, should it keep going on?
if not, you dont need to handle the exception in the sub functions, just let it raise the exception and handle it in main()